code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -61,14 +61,28 @@
 
 /obj/item/gun/energy/e_gun/dragnet
 	name = "\improper DRAGnet"
-	desc = "The \"Dynamic Rapid-Apprehension of the Guilty\" net is a revolution in law enforcement technology."
+	desc = "The \"Dynamic Rapid-Apprehension of the Guilty\" net is a revolution in law enforcement technology. Hit a teleport beacon with it to set the bluespace net teleport destination."
```

Comment:
May want to put this in examine() with notice spans so it stands out from the fluff